source/config/__init__.py

```diff
@@ -10,14 +16,18 @@
 from cStringIO import StringIO
 import itertools
 import contextlib
+from copy import deepcopy
 from collections import OrderedDict
 from configobj import ConfigObj, ConfigObjError
 from validate import Validator
-from logHandler import log
+from logHandler import log, levelNames
+from logging import DEBUG

```

Comment:
This is also accessible as `log.DEBUG` (as opposed to the method `log.debug`). I thought I'd let you know in case you found it nicer to read, but I'm also happy for it to be left as is.